packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-mention.spec.ts

```diff
@@ -1173,14 +1173,16 @@ describe('RichTextEditorMentionListbox', () => {
             expect(pageObject.getSelectedOption()).toBe('user2 name');
         });
 
-        // Intermittent, see https://github.com/ni/nimble/issues/2150
-        xit('should commit mention into the editor when clicked', async () => {
+        it('should commit mention into the editor when clicked', async () => {
             await appendUserMentionConfiguration(element, [
                 { key: 'user:1', displayName: 'username1' }
             ]);
             expect(pageObject.isMentionListboxOpened()).toBeFalse();
             await pageObject.setEditorTextContent('@');
             expect(pageObject.isMentionListboxOpened()).toBeTrue();
+            // Editor loses focus when a mention popup is opened
+            // So focusing the editor before committing the mention node
+            await pageObject.focusEditor();
             await pageObject.clickMentionListboxOption(0);
```

Comment:
Focusing the editor only where the mention options are selected from the mention-listbox and not in the commonplace like `setEditorTextContent()` as other tests are not required to focus on the editor. Let me know if it needs to be changed.

Reply:
Resolving this comment as the right approach is to focus on the page object where we set content and not in the tests like this.